nym-vpn-x/src-tauri/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "nym-vpn-x"
+name = "nymvpn-x"
```

Comment:
Looking at this and @pronebird pointing me to https://github.com/tauri-apps/tauri/issues/9668 I think we shouldn't rename the crate after all. Let's stick to the current one and try to sort out the windows installer in a follow up PR

Reply:
@octol the nsis installer template is not the only impacted area, please consider this:

>Jon sticking with `nym-vpn-x` will have these consequences:
- tauri will use and create `nym-vpn-x` as app dirs (eg config cache .local/state etc) instead of the current and expected `nymvpn-x`
- will need to code refactor to switch to `nym-vpn-x` and drop `nymvpn-x` 
- need to update linux installer scripts
-  need to update aur packages (maybe deb package ?),
- on new app version install, these dirs will be created along the old ones in the user file system
- nsis installer will have to be customized
- a lot of changes required in the gh build workflow as final emitted binaries changed from `nymvpn-x` to  `nym-vpn-x`
(+ I'm probably missing some other impacts)

Ofc in a next iteration, we could handle this global name changing, which is planned anyway

Reply:
Yeah I am not sure if hooks can be used for that, maybe there is another way to rename the installer package if that's the goal we have here. 

Unrelated to this PR: however I think we shouldn't copy `installer.nsh` unless we need heavy modifications to it because upstream fixes to this large file (1000 lines of code) might save us a lot of trouble. I know we will not be manually syncing it. Hooks can be used to add extra scripts if we need to.